packages/create-svelte-ux/bin.js

```diff
@@ -120,8 +121,7 @@ copy(
     PROJECT_NAME: projectName,
     SVELTE_UX_VERSION: version,
   },
-  { '.meta.gitignore': '.gitignore' },
```

Comment:
If that helps with the review: 

For local comparison, to just fix copying the ignore file:

```diff
- { '.meta.gitignore': '.gitignore' },
+ { '.meta..gitignore': '.gitignore' },
```

Then the results of the updated script can be compared with the old fixed script and verified.